src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -210,6 +223,25 @@
                 >
                 <span>{{ workflow?.topicAutomatic }}</span>
               </div>
+              <div *ngIf="!isPublic">
+                <span
+                  ><strong matTooltip="Short description of the {{ entryType$ | async }} generated by AI">AI Generated Topic</strong
+                  >:&nbsp;</span
+                >
+                <span data-cy="topicAI-text">{{ workflow?.topicAI }}</span>
+                <a
+                  *ngIf="workflow?.topicAI"
+                  [href]="Dockstore.DOCUMENTATION_URL + '/faq.html'"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  data-cy="ai-bubble"
+                  class="bubble ai-bubble"
+                  matTooltip="This topic sentence was AI generated. Non-published workflows will not be sent for AI topic sentences."
```

Comment:
We say that unpublished workflows aren't sent for AI topics, but it's possible that we generate an AI topic for a published workflow, then the workflow owner unpublishes it. For this case, should we still display the AI topic? Seems like a waste to hide it, so maybe the tooltip needs to be modified.

Reply:
Might be simpler on both cases if you get rid of the double negative. 
i.e. Instead of
`Non-published workflows will not be sent for AI topic sentences.`
`Only published workflows will be sent for AI topic sentences.`

Reply:
The tooltip now says `AI topic sentences are only generated for published entries.` in non-public facing pages. Screenshots updated.